src/Pages/ohyeonkwon/Login/Login.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import './Login.scss';
+//import { Link } from 'react-router-dom';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      id: '',
+      pw: '',
+      nickname: 'nickname',
+      phone_number: '010-1234-5626',
+      background: '',
+      disabled: true,
+    };
+  }
+
+  goMain = () => {
+    this.props.history.push('/mainH');
+    // fetch('http://10.58.7.108:8000/user/signin', {
+    //   method: 'POST',
+    //   body: JSON.stringify({
+    //     email: this.state.id,
+    //     password: this.state.pw,
+    //     nickname: this.state.nickname,
+    //     phone_number: this.state.phone_number,
+    //   }),
+    // })
+    //   .then(response => response.json())
+    //   .then(result => {
+    //     if (result.token) {
+    //       //성공
+    //       alert('success');
+    //       this.props.history.push('/mainH');
+    //     } else {
+    //       //fail
+    //     }
+    //   });
+  };
+
+  inputValue = e => {
+    // input에 작성한 값
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+    console.log(this.state.id, this.state.pw);
+  };
+
+  colorChange = () => {
+    if (this.state.id.includes('@') && this.state.pw.length > 4) {
+      this.setState({
+        background: 'black',
+        disabled: false,
+      });
+    }
+  };
+
+  render() {
+    return (
+      <div>
+        <title>Westagram</title>
```

Comment:
```suggestion
  render() {
  let validation = this.state.id.includes('@') && this.state.pw.length > 4;
    return (
      <div>
        <title>Westagram</title>
```
이렇게 바꿔서 console.log(validation)을 찍어보세요!
이런식으로 변수로 만들어서 사용하면 colorChange 함수가 필요없어집니다!